### ISSUE TITLE
PreferencesUpdateEvent won't announce on multiple changes

### ISSUE THREAD

**Issue by wallace11** [Ulauncher/Ulauncher#239]
**Steps to reproduce:**

1. Create a `manifest.json` with 2 or more `input` type preferences.
2. Set initial value → Save.
3. Subscribe to `PreferencesUpdateEvent`.
3. Go to settings and update both values → Save.
4. Announced event contains only the first change.

Here's the code I used:
```python
class PreferencesUpdateEventListener(EventListener):
    def on_event(self, event, extension):
        print vars(event)

class Test(Extension):
    def __init__(self):
        super(Test, self).__init__()
        self.subscribe(PreferencesUpdateEvent, PreferencesUpdateEventListener())
```
 `manifest.json` contains two fields: `id1` and `id2`.
Initial values: `{'id1': 'A', 'id2': 'B'}`
Log of changing both values in Settings `id1`→`B`, `id2`→`A`:
```
test | 2018-05-13 12:33:48,872 | DEBUG | ulauncher.api.client.Client: on_message() | Incoming event PreferencesUpdateEvent
{'new_value': 'B', 'old_value': 'A', 'id': 'id1'}
```

**Ulauncher version**: v4.0.7.r19
**Linux distro**: Arch
**Desktop environment**: KDE

**Comment by gornostal** [comment]
Thanks for reporting this bug. I'll look into it

**Comment by gornostal** [comment]
Fixed in 4.0.8.r1. Please reopen if needed
